### DISCUSSION TITLE
Power Amplifier and Antenna Diversity

### DISCUSSION THREAD

**Discussion by iottrends** [ExpressLRS/ExpressLRS#1702] [General]
Hi,
I am  using the DIY_2400_RX module, planning to add a Power Amplifier to boost the signal strength to 100mw(20dbm)

Planning to use Power Amplifier IC AT2401C.

GPIO_PIN_RX_ENABLE 9 --> RXEN Pin on AT2401C
GPIO_PIN_TX_ENABLE 10--> TXEN pin on AT2401C
Questions?
1. Do I need to define this in DIY_2400_RX_ESP8285_SX1280.h?
2.  Anything else I need to change in the config files?
3. How do I define power control? : 10mw, 20mw and 100mw.

**Comment by JyeSmith** [comment]
For that PA you can actually connect RXEN  directly to 3V3.  Then define TXEN in the json or even easier with v3, go to the hardware.html and enter the pin number then save.

Check out the link below for how to setup the output power (power_values) which can also be done from hardware.html.

https://github.com/ExpressLRS/ExpressLRS/blob/master/src/hardware/RX/Generic%202400%20PA.json
